### DISCUSSION TITLE
16.04.6.3 ISO

### DISCUSSION THREAD

**Discussion by student1houston** [Security-Onion-Solutions/securityonion#5347] [Unsupported Versions] [answered]
I was looking for (16.04.6.3 ISO) built: 11/26/2019.
Does this ISO still exist?

**Comment by dougburks** [comment, marked as answer]
Security Onion 16.04 reached End Of Life on April 16, 2021:
https://docs.securityonion.net/en/2.3/eol.html
https://blog.securityonion.net/2021/04/security-onion-1604-has-reached-end-of.html

**Comment by student1houston** [comment]
Thank you for the information.

Maybe you can help me with my downloading issue.



I have VMware Workstation Pro 16.
I am trying to download security onion-ISO for my school project.


  1.  I downloaded (securityonion-2.3.70-GRAFANA) from Github.

  1.  I attempted to download the newer ISO on my VMware and in 55 seconds,
  2.  I receive the message, "not enough space to install security onion.  You need at least 99 GB to proceed."

My Laptop: Windows 10 Home / 8.00 GB Ram
I use an external hard drive with 5 TB, in case of extra storage.

Please assist me with any suggestions or options to get this accomplished.




Sincerely,

Marco



________________________________

**Reply by dougburks** [reply]
You'll need to increase your virtual drive size as described in the VMware Workstation section of our documentation:
https://docs.securityonion.net/en/2.3/vmware.html#workstation-pro

If you have further questions about Security Onion 2, please start a new discussion since it's no longer related to 16.04.6.3.

Thanks!
